Headers/Foundation/NSString.h

```diff
@@ -802,6 +802,11 @@ typedef NSUInteger NSStringEncodingConversionOptions;
 - (const char *)UTF8String;
 #endif
 
+// #if OS_API_VERSION(MAC_OS_X_VERSION_10_9,GS_API_LATEST)
```

Comment:
Why are these guards commented out?

Reply:
Because with them it wasn't seeing the selectors.   I'm not sure what is not defined.   Will come back to this in a bit.

Reply:
Re-added these into the file.  Re-tested and working.